first_run.html

```diff
@@ -0,0 +1,35 @@
+<head>
+  <style>
+    h2 {
+      text-align: center;
+    }
+
+    #content {
+      font-family: sans-serif;
+      margin: 50px auto;
+      max-width: 600px;
+    }
+  </style>
+
+  <title>Chrome Flatpak Beta</title>
+</head>
+
+<body>
+  <div id="content">
+    <h2>Chrome Flatpak Beta</h2>
+
+    <p>Please be aware of the following as you use this:</p>
+
+    <ul>
+      <li><b>Do NOT immediately file bugs on the Chromium bug tracker!</b> File them to the corresponding
```

Comment:
I don't think you would file bugs from Chrome in the Chromium bug tracker, we have a dedicated support for Chrome itself AFAIK.

Reply:
Oh I've definitely seen several people file bugs on the Chromium tracker without using the feedback option :sweat_smile: That being said I should definitely advise against using the feedback option as well for the moment.

Reply:
FYI the feedback option in Chrome just goes to the Chromium tracker. There is no "dedicated space" for one or the other.

Reply:
@Garbee we actually do, but it's for Enterprise concerns, we usually use the Google Help Center for that.

Reply:
Oh interesting. Perhaps what I've seen in the main hose previously was dumps from that when not necessary or just an old method of managing things. Good to know there is some separation there in case it comes up in the future.